### PR TITLE
net-dns/bind-9.13.0 and net-dns/bind-tools-9.13.0 version bump

### PR DESCRIPTION
**Pull request by candrews**
https://lists.isc.org/pipermail/bind-announce/2018-May/001092.html

Remove the unused seccomp use flag and unused  --disable-openssl-version-check and --with-randomdev configure options. Also bumped to EAPI=6

### CODE REVIEW

File: net-dns/bind/bind-9.13.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+
+MY_PV="${PV/_p/-P}"
+MY_PV="${MY_PV/_rc/rc}"
+MY_P="${PN}-${MY_PV}"
+
+SDB_LDAP_VER="1.1.0-fc14"
+
+RRL_PV="${MY_PV}"
+
+NSLINT_DIR="contrib/nslint-3.0a2/"
```
Comment (Dessa) [inline net-dns/bind/bind-9.13.0.ebuild]: this can go away entirely - it has been removed in 9.12
https://bugs.gentoo.org/654206

Comment (candrews) [reply net-dns/bind/bind-9.13.0.ebuild]: Thanks! I've updated the PR accordingly.

**Comment by candrews** [comment]
@idl0r Is there a downside?

I feel like having development builds in portage ~keyworded is exactly what the ~keywording is for :)

**Comment by idl0r** [comment]
~keyword doesn't mean unstable or something like that, just testing. It is often used and expected as "stable" anyway. I'd rather hardmask those packages, really for testing purpose. I never added development builds until now. It is very likely to contain more bugs than the stable release do.

**Pull request closed without merging**